ext_test.go

```diff
@@ -35,9 +34,8 @@ func TestGetFailures(t *testing.T) {
 	d := NewDHT(ctx, hosts[0], tsds)
 	d.Update(ctx, hosts[1].ID())
 
-	// Reply with failures to every message
 	hosts[1].SetStreamHandler(ProtocolDHT, func(s inet.Stream) {
-		s.Close()
+		// hang forever
```

Comment:
how is this a good idea?

Reply:
then again it's just a test...

Reply:
The test is supposed to test a timeout. I'm not sure why it worked before as it *should* have searched all peers, determined that none of them were going to respond, and returned NotFound to the user (or at least that's what happened after my change).